views/js/test/tools/upload/test.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <title>Test - Upload speed Tester</title>
+        <base href="../../../../../../tao/views/" />
+        <link rel="stylesheet" type="text/css" href="js/lib/qunit/qunit.css">
+        <script type="text/javascript" src="js/lib/require.js"></script>
+        <script type="text/javascript" src="js/lib/qunit/qunit.js"></script>
```

Comment:
Not mandatory, but it should be better to add code coverage using blanket.

Also, there is no grunt config to run CLI tests. If you have some time could you add this file ?